aiosmtpd/handlers.py

```diff
@@ -80,7 +79,7 @@ def process_message(self, peer, mailfrom, rcpttos, data, **kws):
             add_separator = True
         if add_separator:
             print(file=self.stream)
-        self._print_message_content(peer, data)
+        self._print_message_content(session.peer, message.received_data)
```

Comment:
Yeah, here, I wonder if we really need to separate this into two different objects.  E.g. is it more or less confusing if `received_data` were on the `session` object?  Correct me if I'm wrong, but I believe that sessions and messages have the same lifetime, and both get reset on a `RSET`.

I kind of like the elegance of just passing around a session object.